Client/mods/deathmatch/logic/CClientVectorGraphic.h

```diff
@@ -0,0 +1,53 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#pragma once
+
+#include <svgdocument.h>
+#include "CClientVectorGraphicDisplay.h"
+
+using namespace lunasvg;
+
+constexpr float DPI = 96.0;
```

Comment:
Make it a class constant of `CClientVectorGraphic`. Don't introduce a global constant.

Reply:
Added to the class